applications/ContactStructuralMechanicsApplication/custom_conditions/ALM_frictional_mortar_contact_condition.cpp

```diff
@@ -157,7 +157,7 @@ void AugmentedLagrangianMethodFrictionalMortarContactCondition<TDim,TNumNodes,TN
                 // Integrating the mortar operators
                 for ( IndexType point_number = 0; point_number < integration_points_slave.size(); ++point_number ) {
                     // We compute the local coordinates
-                    const PointType local_point_decomp = integration_points_slave[point_number].Coordinates();
+                    const auto local_point_decomp = PointType{integration_points_slave[point_number].Coordinates()};
```

Comment:
Hi Riccardo, I checked it and yes it works. This could be cleaned up more with `const auto& local_point_decomp = integration_points_slave[point_number];`, but aim of this PR was to add the explicit and replace broken implicit conversions with explicit conversions (essentially not changing the behavior/performance). I could go ahead and clean these up if you want, but I would prefer the application developers do this (ideally in subsequent PRs) since I'm not familiar with this app (or many other apps). Also there is code repetition which I think is being generated by some automated tool. It's a bit of effort to fix all of these by hand.

Reply:
ok